WooCommerce/Classes/Authentication/Store Creation/Free Trial/FreeTrialSurveyCoordinator.swift

```diff
@@ -0,0 +1,25 @@
+import UIKit
+
+final class FreeTrialSurveyCoordinator: Coordinator {
```

Comment:
Nit: A comment explaining this class would be helpful.

Reply:
Done as part of https://github.com/woocommerce/woocommerce-ios/pull/10319/commits/fd5c8beda1db35ee4fd530d76fc6215399cc7b47